server/src/main/java/at/htl/franklyn/control/ExamineeRepository.java

```diff
@@ -0,0 +1,51 @@
+package at.htl.franklyn.control;
+
+import at.htl.franklyn.entity.Examinee;
+import jakarta.enterprise.context.ApplicationScoped;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+public class ExamineeRepository {
+    // Key: Ip address, Val: Examinee instance
```

Comment:
The comment about the key value being the Ip address is fine  but I would not write the  value as well since you can already see it in the type signature.